readme.md

```diff
@@ -124,5 +156,18 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
+A common git work-flow would be:
+1. checkout development from master
+2. checkout feature branch, add feature
+3. checkout development, merge feature, testing
+4. checkout master, merge development, testing
+5. (re)deployment
+
+Another one might be:
+1. fork repo
+2. checkout personal branch
+3. make changes, add and commit changes
+4. push to origin (personal branch)
+5. open pull request to upstream, probably to development branch
 
 ```
```

Comment:
:tada: (10/10) Great job on this! Feel free to check out a solution if interested, otherwise let me know if you have any specific questions.